docs/proposals/35-observability2.0-victoria-logs.md

```diff
@@ -0,0 +1,188 @@
+gep-number: 35
+creation-date: 2025-10-13
+status: implementable
+authors:
+- "@nickytd"
+- "@rrhubenov"
+reviewers:
+- "@plkokanov"
+---
+
+# GEP-35: Migrating from Vali to VictoriaLogs
+
+## Table of Contents
+
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+- [Alternatives](#alternatives)
+
+## Summary
+
+This proposal introduces the deployment of [VictoriaLogs](https://github.com/VictoriaMetrics/VictoriaLogs) which will act as the replacement for [Vali](https://github.com/credativ/vali) in the Control Plane of all Shoot clusters that have the observability stack enabled, as well as in Seed clusters and Garden clusters.
+After the work that has been done on [GEP-34](https://github.com/gardener/gardener/pull/11861), we are in a favourable position to easily switch our storage layer for the clusters' log signals.
+
+A new operator is introduced that will be deployed to the `garden` namespace of seed & garden clusters.
+This operator will manage the deployment of the new `VictoriaLogs` instances to the `garden` namespace of seed & garden clusters, as well as to the Control Plane namespace of Shoot clusters via the `VLSingle` CustomResource.
+
+Since we'd like to make the transition from `Vali` to `VictoriaLogs` as painless as possible, this proposal also introduces multiple ways to migrate in already running landscapes without losing data that has accumulated with the existing `Vali` instances.
+
+## Motivation
+
+Since version v2.3.0, [Loki](https://github.com/grafana/loki) has switched from the Apache-2.0 license to a significantly more restrictive AGPLv3 licene.
+Since then, the Observability stack of Gardener has been using [Vali](https://github.com/credativ/vali) -- a fork of Loki 2.2.1, which is the last official version that maintains the Apache-2.0 license.
+
+However, the fork maintains only security updates, thus leading to no new features or improvements getting integrated.
+As such, an upgrade is due so that we can benefit from new technologies and optimizations in the world of log databases.
+
+One such advancement is [OTLP](https://opentelemetry.io/docs/specs/otel/protocol/), discussed in more detail in [GEP-34](https://github.com/gardener/gardener/blob/master/docs/proposals/34-observability2.0-opentelemtry-operator-and-collectors.md).
+ After careful consideration, we've converged to using [VictoriaLogs](https://github.com/VictoriaMetrics/VictoriaLogs) as the default backend for storing logs.
+
+### Goals
+
+- Deploy the VictoriaOperator & VictoriaLogs deployment in seed clusters.
+- Deploy the VictoriaOperator & VictoriaLogs deployment in the garden cluster.
+- Deploy a VictoriaLogs deployment to the control plane of new & existing `Shoot` clusters.
+- Give an appropriate migration strategy from Vali to VictoriaLogs for existing clusters.
+
+### Non-Goals
+
+- Provide tooling around the migration from `Vali` to `VictoriaLogs`
```

Comment:
Not sure I understand this. We want to have a migration strategy, but not provide tooling. It seems to me that having a migration strategy might require some tooling.
Please explain.